### PR TITLE
Feature: adding checkstyle dependency to project

### PR DESCRIPTION
**Pull request by eapg**
* Adding checkstyle dependency to java etl project

### CODE REVIEW

File: checkstyle-suppressions.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+        "-//Checkstyle//DTD SuppressionFilter Configuration 1.2//EN"
+        "https://checkstyle.org/dtds/suppressions_1_2.dtd">
+
+<suppressions>
+
+</suppressions>
```
Comment (reynoldmorel) [inline checkstyle-suppressions.xml:9]: Add enter here

File: checkstyle.xml [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+        </module>
+        <module name="SuppressWithNearbyCommentFilter">
+            <property name="commentFormat" value="CHECKSTYLE.SUPPRESS\: ([\w\|]+)"/>
+            <!-- $1 refers to the first match group in the regex defined in commentFormat -->
+            <property name="checkFormat" value="$1"/>
+            <!-- The check is suppressed in the next line of code after the comment -->
+            <property name="influenceFormat" value="1"/>
+        </module>
+    </module>
+</module>
```
Comment (reynoldmorel) [inline checkstyle.xml:377]: Add enter here

**Review by reynoldmorel — CHANGES_REQUESTED** [review]

File: checkstyle-suppressions.xml (hunk 1 above)
Comment (eapg) [reply checkstyle-suppressions.xml]: done.

File: checkstyle.xml (hunk 2 above)
Comment (eapg) [reply checkstyle.xml]: done

**Review by reynoldmorel — APPROVED** [review]

**Pull request merged**